src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -47,6 +48,29 @@ public void storeMessage(Message message) {
     datastore.put(messageEntity);
   }
 
+  public void deleteMessageWithID(String messageID, String user) {
+    Query query =
+        new Query("Message")
+            .setFilter(new Query.FilterPredicate("ID", FilterOperator.EQUAL, messageID));
+
+    PreparedQuery results = datastore.prepare(query);
+    Entity messageEntity = results.asSingleEntity();
+
+    if (messageEntity == null) {
+      System.err.println("Invalid Message ID - " + messageID);
+      return;
+    }
+    if (!messageEntity.getProperty("user").equals(user)) {
+      System.err.println(
+          "Invalid Credentials: User "
+              + user
+              + " attempt to delete message by "
+              + messageEntity.getProperty("user"));
```

Comment:
Are these debugging statements? Is there ever a case in which these will be true if the delete button only exists for that user's messages for existing messages? I'm asking because I'm not sure